src/pages/index.tsx

```
@@ -25,6 +28,45 @@ const Home = () => {
+  const { t } = useTranslation()
+  const characterPresets = [
+    { key: "characterPreset1", value: settingsStore((s) => s.characterPreset1) },
+    { key: "characterPreset2", value: settingsStore((s) => s.characterPreset2) },
+    { key: "characterPreset3", value: settingsStore((s) => s.characterPreset3) },
+    { key: "characterPreset4", value: settingsStore((s) => s.characterPreset4) },
+    { key: "characterPreset5", value: settingsStore((s) => s.characterPreset5) },
+  ];
+
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if ((event.metaKey || event.ctrlKey) && event.shiftKey ) {
```

ESLint

[error] Delete `·`

(prettier/prettier)